### ISSUE TITLE
It would be good to be able to cap the validation set size for intra-run evaluation to keep timing down when large validation sets are used.

### ISSUE THREAD

**Issue by mmcdermott** [mmcdermott/MEDS_EIC_AR#39]

**Comment by mmcdermott** [comment]
It also recommends:

```
validation_config = {
    "limit_val_batches": 200,      # ~200 batches for monitoring
    "val_check_interval": 1000,    # Validate every 1000 training steps
    "check_val_every_n_epoch": 1,  # Validate every epoch
}

# This gives you ~5-10 minutes of validation instead of hours
trainer = Trainer(**validation_config)
```
